tf-modules/cumulus-test-cleanup/reap-kinesis.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+// handy script to delete any old test kinesis streams that are created but not
+// cleaned up.
+
+const aws = require('aws-sdk');
+const moment = require('moment');
+const kinesis = new aws.Kinesis();
+
+const deleteOlderThanDays = 1;
+
+const getStreams = async () => {
+  return kinesis.listStreams({})
+    .promise()
+    .then((result) => result.StreamNames);
+};
+
+
+const filterOld = (streams) => {
```

Comment:
Node.js question that's come up elsewhere - is there a reason this isn't just a function definition?

Reply:
haha idk ask @flamingbear. I can change it if you want though.

Reply:
I have a *very* mild preference for readability reasons, but not worth changing if there's a reason it's that way.

Reply:
I changed it https://github.com/nasa/cumulus/pull/1452/commits/5a14139d7aa3e97b5fd1b21edff9e3a010c112d0